AEPServices/Sources/ui/UIUtils.swift

```diff
@@ -0,0 +1,32 @@
aWM0T0ewfKNJzJqPIsvbxQ6cfAu7+WAHYN3x31Qjhw/2xkEHVuqjarK4XrtTvhG2IPIUD9A4ef6owIOP+VuGq7PwqVu5ONU6t9Zd3174E577frgWIIqdtcAqsyprKg7HFQW+dBqQArzNE1wMR/V4sD5liNGKiK6RirBLfxO9fiPsmW4XX6zDP9x3KfBxp+E7HU6FQhDqP69SNK3L8DmWZY7Khjl7CXtSS4BKOqFXvqmC1mm9BnKfzEfpfOyseqTd3vs+qjjffiHKoEPNSaUnfujruX2dxqoLMSskvSW7PZ8rsGV0w2hnooYAVyjyAdjKPUWEUbBfimVFt0jgVeqqqB7XAbXuh/c63EoVU/ivZwwOBhWlwxmAvhp3eVNL5jMN36xK1lQ+exZCnyE58DBz0M3tUeVAZzdDSyhl6LqQjdll0jdxZJiPkgPRrP7y6CS7p1WKXcqsGxlKiqnqe0l13Hx2aijRjmY3As7AMbXuJS6WkElLkPgH4PLr2J0cwZd5Fnoak6mUonekcEyzfDLt7sayGjgnLWmHRyFDyY+EgVzPrTLPNGHcn8POcnuzi7NTQ7qpVyxG7tp5FJyKLtJV5kQpgmoANOdCSoL1beEvrijTQNgLbtGhM1Wk3s5chRg5MPLLnIplb2CClrKO2W0VXCw09evX/8QvuJ7ALZAWKObvEuBbBiYeBfdnXVO2SYQzC/mQ/sGHOpC6mQnH5kn6u0FWAEogqmn2i+geBfHNxLPNXGZSwkmOdWdyqVML5sfG7DMQANc5kPOpY0+II2wGWOMr9SzIGK+kbABMgEGeBxByslG0Ms4vDyB7vGBdAX3QaBdbzepySCr07H9wr/3xvter0N6niOdEJ7NjMzJhEyfcmyDQ13gmZ90/Vcy1xoH3BFmL5en6DGVEOdUqZPyiINQIQAuD0KTk8mlMqB+LFyK9Dumf013Xt85jtzwpGZ+vKJ7RFhV9hpVPSHAAErIYLjV3i+hwgXZUuKP8nWP4T4cYN0N7ifATuVaQ5gohI3HpdQfgK4b/w4rv+mQBlZlmx5XRxbUnzzfApe5hGmtC3D/iLxuJ8+e9+qvZuIIIQC0j3Kpi4pTXwSU6gPMFIOZDKgsRwmoO2XUF5U65HaHVeyQNPu+oqJzXrSLZvdI5atYHaUyx+BOAkiALeLLdiQZo8kaYG6ibSGTLOb9Br76k2iou/NVn0BXeYQ0mokTx8cAm46cB0HQ7zk8q3DhJFyH1RG30+3U8cFGBeocjw9kiutG8ufhwD+ES11O2K106fFLGdxxMjW09nHtp6yhMYfOAcrqb9zcbT4ll7Lbo8vsGvGZTjNPOd0fh50b88eRdt80bVy2cvMvylcntvS3zjzzTPMYHIx5xwCYXEyQYcYRWANkPDsPGUAj/GI4mgBs0RAezruKL/4owLWRRtlEQx5F3MMdEGD3zUf2FDBeTUM6NmDTa5ab/KCwn4h/KAOkLVxXcW1GFtpC4J2APP4DCQdS2HZhZFzkR2Wn+/oc4eXD+gVk3uAQgjl1469y2257p67uElyqWQnDa6cZ8k52CzCSzPkTWLrdbjZC5Y/QSKMAIwAmiDQAye7SOcdFGWi6aSh/VaXGv0VjC7TlAG011zpobOBaj3stJFYiyyzjUt3B8oQ0ju06MeUhHkNuE2gaFyj6p1E3e+4lPUG0C+DsII1tXFtx38SVNyzYhcW/tglvF2Y8OZUz1k7FyK1CEBds+Ls5ZYqPwL2b0+jxmiO9YLXEu8sBijyFyQ2Xn+9SNhWf85ftABicq4WOxnVHIzRCgIyGCJABCkEkyDLQnPgWbOP4L+L9GDgZ4X2LBmtpAzifZV2GC65HuOCA0B/D7YAi58HJcX+LFn9FwymH8XAUJh2T4BAUnn7Y9uLZrtYonpAniDwkOMLVYNLff9P4dZClyQAT9KcRyqcpa5uyZhmqAyjo3GW6PfPSa7pkjt6XP+5nGiD6IBomZyUqlikh/rAb8oVfsVxIYbdF44xijcCBRvALwZ8GGsVfcDglNMb7Ud28HxM0PsMVBJbgCaCZGM+FeCJunMtZYSv0ZSxaHUtwn1h9QfzIHLZ5E0gu+fGSK3VAg3uKfAWAoKW/v73Vr/oBNw9wTX/W0EY/F9wJOZOfbOyOv4QQXxNp5XqpvO6a94q/3zXz1smVFVlf8eV+5StfUWi18r38S8IEQ3W5gCOnfQjG+xDand13z5zru3fifyuN5lbyHdg3478dextg2wrItvL+Rmh5cJwLqrZj30Rbx+krgGgH8Yx7i27o7fCdYeqwxgka0pKmtOs03Ehimi64c33UTvNk3syjeTXPXPstg2WBVoz2LKNl5dnyx0H92J2vDP+7rMns/S6bwUEZo/Fy3pu27oLuowUAWky+FnSXLb56u8wWQGhhj9CIdn9yunD7rKHRonvE7hjCFACBfbOTkQ4g8bd7rJxRvTZj8LfLAy8zhjCeNh0edm2YNnb8o4huri3nJR/Nrk7QZOD8xHCnfm2UG6Xfu58YPxo0l6Np644LZauA4peA9nytgm60hUK1AHAtlKr+QVRQhWGUGW7D4eZ/TUujHnypQyhpM3hIjNzyXGWJW2CFYeTZhvuUqDvaKDxL9GqCK/5cC7DKjfx3W1I9l2R/osFkAbLJDZGff+JtWjaXKWyOji74X7HlUjdWAAA5UCHQvOBioQGvGFYFnqYbecc5u0wz6gcwsi6QhYEYkSRomm6f4U6lIPKCw3Hs+76Sri3A5E/d/YdyEPgpAlNdnrC6Kqz54qfF3QCaRcrlLdADFeiBkmCjwdPFF1/M/8lPS3SjEUbQzacOBI7hGLmV0EjQ8B9ESRCRRiKNAFNNK4PpLj2im0+55wpzlx4VzpX5+bynB+s0JuFlMAJG5WJEF1SY4qKLLhJY5fnnnx+AqrlcEoARsOdWAybJfTQCK9OoOZOginfeevLR8f9pdfSttJ/Wws6nXD0cbj5RIsx/N+DMVTH/H12VRhzaJ53wAAAAAElFTkSuQmCC"
```

Comment:
Can we put this at the top of the file?

Reply:
Its too big and the function will move way down in the file because of this 
so kept it at the bottom